src/bootlint.js

```diff
@@ -964,6 +966,86 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
```

Comment:
Understandable. At least to start I just wanted to get it fully working with a full set of tests before looking into refactoring. I'll get to that tomorrow though. It's late here on the east coast :smiley: